src/styles/variables.scss

```diff
@@ -0,0 +1,5 @@
+$calendar-day-height: 100px;
+$calendar-day-width: 100px;
+$days-of-week: (mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6);
+$month-lenth: ("28", "29", "30", "31");
```

Comment:
The values in the 'month-length' list are strings, but they should be integers. Update to [28, 29, 30, 31].